drivers/ds323x/Makefile

```diff
@@ -0,0 +1,4 @@
+# enable submodules
+SUBMODULES := 1
```

Comment:
So far you just implemented *ds3234.c*. Do we actually need the SUBMODULES now?

Reply:
Implemented this way makes it easy to add a ds3231.c later on